bridge/discord/discord.go

```diff
@@ -381,6 +381,19 @@ func (b *Bdiscord) webhookSend(msg *config.Message, webhookID, token string) (*d
 		err error
 	)
 
+	// If avatar is unset, maybe we need to set the local avatar
+	if msg.Avatar == "" {
+		for _, val := range b.GetStringSlice("UseLocalAvatar") {
+			if msg.Protocol != val {
+				continue
+			}
+			if avatar, ok := b.findAvatar(msg); ok {
+				msg.Avatar = avatar
+			}
+			break
```

Comment:
Isn't this break redundant ?

Reply:
I think it needs to be one line up

Reply:
No it's fine as it is, but it's confusing, so I will refactor this chunk